common/models/change.js

```diff
@@ -110,7 +112,7 @@ module.exports = function(Change) {
           })
           .join('\n');
 
-        var msg = 'Cannot rectify ' + modelName + ' changes:\n' + desc;
+        var msg = g.f('Cannot rectify %s changes:\n%s', modelName, desc);
```

Comment:
There was no space there to begin with.
Also, `desc` will be displayed on a new line.